src/extraction/fathom/ruleset_factory.js

```diff
@@ -275,40 +260,54 @@ export default class RulesetFactory {
        */
       // consider all visible img elements
       rule(dom('img').when(this.isVisible.bind(this)), type('imageish')),
+      // and divs, which sometimes have CSS background-images
+      // TODO: Consider a bonus for <img> tags.
+      rule(dom('div').when(fnode => this.isVisible(fnode) && this.hasBackgroundImage(fnode)), type('imageish')),
```

Comment:
Do we actually successfully pull the background image from these elements? I think the wrapper code around this factory assumes image matches are `img` tags and pull their `src`.

Reply:
Good question! I had tunnel vision and didn't look beyond upping the recognizer's accuracy. Let me see if I can figure it out.